### ISSUE TITLE
Editions statements grammatically inseparable information

### ISSUE THREAD

**Issue by mjmascaro** [rbms-bsc/DCRMR#38]
In RDA, designation of edition is the only edition statement element that has instructions regarding how to record that information if it integral part of another element.  For that reason, DCRM(B) 2B6. Edition statements that are grammatically parts of other areas was revised to the designation of edition specific in the RBMS PS.  The inverse DCRM(B) instruction 2B8 Edition statements with grammatically inseparable information was also mapped to designation of edition in the PS (though the wording not updated to be specific to designation of edition).  (See P[S drafts](https://docs.google.com/document/d/1rEVRvlWDJWmWS2G6s7Jrik0_Rv28OQEFSp-mezTmkic/edit#bookmark=id.pdu2ohatvv5d))  

This mapping of 2B8 to designation of edition was carried over into [DCRMR](https://docs.google.com/document/d/1RdsSOlT6xC6qTydK7dJvNiZUxUFYXDeUPFbe3a_u-zQ/edit#bookmark=id.ox0636aym6td), with the suggestion that edition statement be replaced with designation of edition.  Looking at the instruction, I think it really applies to the edition statement as a whole and should be moved to the DCRMR section for edition statement.

Q1: Is there general agreement that the guidance in DCRM(B) 2B8 (edition statements with grammatically inseparable information) is relevant to the whole edition statement and mapped to edition statement element.
 Q2: Is the inverse instruction (2B6 edition statements that are grammatically inseparable parts of other areas) also applicable to the whole edition statement and should be mapped there? (This one I am not as clear on).

**Comment by lizoknj** [comment]
I agree with moving both these instructions up to the edition statement element. When I did the first draft of the Edition chapter,  I moved them into the RDA Designation of edition element because that is the RDA equivalent of DCRM(B) edition statement (DCRM(B) 2B, edition statement + DCRM(B), 2C Statement of responsibility relating to edition = Chapter 2, edition area). The instructions on grammatically inseparable and separable edition statements appeared only in 2B, but I think it is reasonable to assume that they apply to the statement of responsibility as well as the designation of edition, and that they should move up to edition statement.

Thinking a little further along those lines, I wonder whether DCRM(B) 2C1.3, which is currently in the RDA SOR relating to edition element, might also be moved up to edition statement, since it involves making a decision on which sub-element within the edition statement the information belongs in.

“2C1.3. In determining the extent of the edition statement and the beginning of the statement of responsibility relating to the edition, it may be necessary to take into account the layout, punctuation, and typography of the title page as well as the sense of the text. Such words as “Revised and enlarged,” when appearing with the name of a person or body, might be transcribed either as part of the edition statement or as part of the statement of responsibility relating to the edition, depending on their presentation on the title page.”

Just a thought.
